### PR TITLE
#267: Update dependencies

### PR DESCRIPTION
**Pull request by jowerner**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -554,7 +549,7 @@
         <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>10.3.1</version>
+            <version>10.3.4</version>
```
Comment (h-arlt) [inline pom.xml:552]: There is an update available: 10.5.0

**Review by h-arlt — COMMENTED** [review]
See comments inline.

**Comment by h-arlt** [comment]
Some other potential updates:
* dnsjava -> 3.5.2
* tyrus-container-grizzly-server -> 1.20
* jaxb-core + jaxb-impl -> 4.0.1

File: pom.xml [hunk 2]
```diff
@@ -219,7 +219,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-ec2</artifactId>
-            <version>1.12.279</version>
+            <version>1.12.326</version>
```
[the remark this answers is not in the document]
Comment (jowerner) [reply pom.xml:231]: Updated to latest.

File: pom.xml [hunk 3]
```diff
@@ -230,12 +230,7 @@
         <dependency>
             <groupId>com.google.apis</groupId>
             <artifactId>google-api-services-compute</artifactId>
-            <version>v1-rev20220720-1.32.1</version>
-        </dependency>
-        <dependency>
-            <groupId>com.google.http-client</groupId>
-            <artifactId>google-http-client-jackson2</artifactId>
-            <version>1.42.2</version>
+            <version>v1-rev20221011-2.0.0</version>
```
[the remark this answers is not in the document]
Comment (jowerner) [reply pom.xml:242]: Updated to latest.

File: pom.xml [hunk 4]
```diff
@@ -338,7 +333,7 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
-            <version>4.1.1</version>
+            <version>4.1.2</version>
```
[the remark this answers is not in the document]
Comment (jowerner) [reply pom.xml]: Updated to latest.

File: pom.xml [hunk 5]
```diff
@@ -372,7 +367,7 @@
         <dependency>
             <groupId>net.bytebuddy</groupId>
             <artifactId>byte-buddy</artifactId>
-            <version>1.12.2</version>
+            <version>1.12.7</version>
```
[the remark this answers is not in the document]
Comment (jowerner) [reply pom.xml]: Updated to latest.

File: pom.xml [hunk 6]
```diff
@@ -497,14 +492,14 @@
         <dependency>
             <groupId>net.sourceforge.pmd</groupId>
             <artifactId>pmd-java</artifactId>
-            <version>6.46.0</version>
+            <version>6.50.0</version>
```
[the remark this answers is not in the document]
Comment (jowerner) [reply pom.xml:518]: Updated to latest.

File: pom.xml [hunk 7]
```diff
@@ -67,8 +67,8 @@
         <maven.compiler.target>11</maven.compiler.target>
         <asm.version>9.3</asm.version>
```
[the remark this answers is not in the document]
Comment (jowerner) [reply pom.xml]: Done.

File: pom.xml (hunk 1 above)
Comment (jowerner) [reply pom.xml:575]: Updated to latest.

**Review by h-arlt — APPROVED** [review]

**Pull request merged**